boards/makerbase/mks_canable_v20/mks_canable_v20_defconfig

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2024 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_GPIO=y
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_ENTROPY_GENERATOR=y
```

Comment:
Why is this enabled?

Reply:
Good catch - copy-paste from other board.